### PR TITLE
Update Request Filters for 0.13

### PR DESCRIPTION
**Pull request by PineappleIOnic**
## What does this PR do?

Update the Request Filters to provide backwards compatibility with 0.12 SDK's

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes, I have

### CODE REVIEW

File: src/Appwrite/Utopia/Request/Filters/V13.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+namespace Appwrite\Utopia\Request\Filters;
+
+use Appwrite\Utopia\Request\Filter;
+
+class V13 extends Filter
+{
+    // Convert 0.12 params format to 0.13 format
+    public function parse(array $content, string $model): array
+    {
+        switch ($model) {
```
Comment (Meldiron) [inline src/Appwrite/Utopia/Request/Filters/V13.php:12]: I think we need one for function entrypoint, right? It was 'code', now it's 'entrypoint'. Wasnt it renamed?

Even if name wasn't changed, we should probably split by space and use the last thing.

File: src/Appwrite/Utopia/Request/Filters/V13.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            // Replaced Types
+            case "database.createFloatAttribute":
+                $content = $this->convertStringToNum($content, "min");
+                $content = $this->convertStringToNum($content, "max");
+                $content = $this->convertStringToNum($content, "default");
+                break;
+            case "database.createIntegerAttribute":
+                $content = $this->convertStringToNum($content, "min");
```
Comment (Meldiron) [inline src/Appwrite/Utopia/Request/Filters/V13.php:20]: I am not sure if these were changed, please double-check

**Review by Meldiron — CHANGES_REQUESTED** [review]

File: src/Appwrite/Utopia/Request/Filters/V13.php (hunk 2 above)
Comment (PineappleIOnic) [reply src/Appwrite/Utopia/Request/Filters/V13.php]: They were changes however double-checking it appears they changed the opposite way. From string to number. I've just fixed this.

File: app/controllers/general.php [hunk 3]
```diff
@@ -45,6 +46,9 @@
     $requestFormat = $request->getHeader('x-appwrite-response-format', App::getEnv('_APP_SYSTEM_RESPONSE_FORMAT', ''));
     if ($requestFormat) {
         switch($requestFormat) {
+            case version_compare ($requestFormat , '0.13.0', '<') :
+                Request::setFilter(new V13());
```
Comment (christyjacob4) [inline app/controllers/general.php:50]: Let's use a prefix like `RequestV12` `RequestV13`

**Pull request merged**